src/settings-ui/Settings.UI.UnitTests/Settings.UI.UnitTests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>net7.0-windows10.0.19041.0</TargetFramework>
     <IsPackable>false</IsPackable>
-    
+    <RuntimeIdentifiers>win10-x64;win10-arm64</RuntimeIdentifiers>
```

Comment:
Just curious, what's the reason for adding it?

Reply:
I've added it. Release CI was erroring on trying to use arm and arm-aot for this project. Was not sure if it was a build racing condition, but this should be pretty harmless otherwise.